webservices/resources/sched_d.py

```diff
@@ -83,4 +83,7 @@ def build_query(self, **kwargs):
     def args(self):
         return utils.extend(
             args.paging,
+            args.make_sort_args(
+                default='load_date',
+            )
         )
```

Comment:
Please add a new line to the end of the file

Reply:
Sure. I just update the file. Please check.